src/sip/transp.c

```diff
@@ -1773,9 +1841,10 @@ static bool debug_handler(struct le *le, void *arg)
 	const struct sip_transport *transp = le->data;
 	struct re_printf *pf = arg;
 
-	(void)re_hprintf(pf, "  %J (%s)\n",
+	(void)re_hprintf(pf, "  %J (%s) (%slistening)\n",
 			 &transp->laddr,
-			 sip_transp_name(transp->tp));
+			 sip_transp_name(transp->tp),
+			 sa_port(&transp->laddr) ? "" : "not ");
```

Comment:
Maybe this function should be adapted in general. Currently, if no listening is open for a transport, this function will print e.g.

`172.21.0.1:0 (TCP) (not listening)`

Not sure if that makes sense since there is no socket option. Maybe we should print something like:

transports:
  UDP, TCP

transport sockets:
  172.21.0.1:5060 (TCP)

and only list actually open sockets

Reply:
Why then print "(listening)" on each line?

Reply:
Done. Removed the "(listening)".